### PR TITLE
Add parameter for custom login title + clear cookies method

### PR DESCRIPTION
**Pull request by Reminouche**
- Add parameter for set custom login title.
- Add logout method for clear cookies. Actually, if a user logs in with oauth (like facebook) and logs out, with the second authentication he can't change his email/password (because it is cached in cookies).

### CODE REVIEW

File: src/Microsoft.WindowsAzure.MobileServices.Android/Authentication/MobileServiceUIAuthentication.cs [hunk 1]
```diff
@@ -17,13 +17,22 @@ public MobileServiceUIAuthentication (Context context, IMobileServiceClient clie
 
         private Context context;
 
-        protected override Task<string> LoginAsyncOverride()
+        protected void Logout()
```
Comment (fabiocav) [inline src/Microsoft.WindowsAzure.MobileServices.Android/Authentication/MobileServiceUIAuthentication.cs]: This is not currently called anywhere. Is the intention to expose this so it is invoked when `LogoutAsync` is called on he `MobileServiceClient`?

Comment (Reminouche) [inline src/Microsoft.WindowsAzure.MobileServices.Android/Authentication/MobileServiceUIAuthentication.cs]: Yes exactly

Comment (fabiocav) [inline src/Microsoft.WindowsAzure.MobileServices.Android/Authentication/MobileServiceUIAuthentication.cs]: If we're adding this here, we'd need to hook it all up so it is properly called. I would break this change into a separate PR if you'd like to have that done, that way we don't need to block on this for your other changes, since they look good and could be merged in.

Comment (fabiocav) [inline src/Microsoft.WindowsAzure.MobileServices.Android/Authentication/MobileServiceUIAuthentication.cs]: Hi, @Reminouche, just following up on this since I haven't heard back.

Comment (Reminouche) [inline src/Microsoft.WindowsAzure.MobileServices.Android/Authentication/MobileServiceUIAuthentication.cs]: Hi. It's ok for me.

Best regards

Le 3 août 2016 20:30, "Fabio Cavalcante" <notifications@github.com> a
écrit :

**Comment by adrianhall** [comment]
Hello @Reminouche - per the discussion with Fabio (and apologies the the delay), we'd like you to split this into two separate PRs.  I don't have a problem approving the title for login and we can discuss the logout code separately.

I'm closing this PR now, but please do re-submit the PR per the discussion.
